src/Axiom/Set.agda

```diff
@@ -164,6 +164,9 @@ record Theory {ℓ} : Type (sucˡ ℓ) where
   ∈-fromList : ∀ {a} → a ∈ˡ l ⇔ a ∈ fromList l
   ∈-fromList = proj₂ $ listing _
 
+  listToFinSet : List A → FinSet A
```

Comment:
This is unused, and I don't think we'll ever need it. Or do you have a use-case in mind?

Reply:
I found a different way of solving the problem, so I can just get rid of this.